ui-tests/src/test/java/org/openmrs/reference/page/EditPatientRelationshipPage.java

```diff
@@ -0,0 +1,34 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Select;
+
+public class EditPatientRelationshipPage  extends Page {
+
+    private  HomePage homePage;
+    private  static  final By SELECT_RELATIONSHIP_TYPE = By.id("relationship_type");
+    private static final By NAME = By.cssSelector("button, input");
+    private static final By NEXT_BUTTON = By.id("next-button");
+    private static final By CONFIRM_BUTTON = By.id("registration-submit");
+
+    public EditPatientRelationshipPage(RegistrationSummaryPage parent) {
+        super(parent);
+    }
+
+    public void clickOnSelectRelationshipType() throws InterruptedException{
+        clickOn(SELECT_RELATIONSHIP_TYPE);
+        Select relationshipType = new Select(driver.findElement(By.id("relationship_type")));
+        relationshipType.selectByVisibleText("Doctor");
```

Comment:
next thing is select a person, ensure your test creates that person in before and removes him and the relationship in after

Reply:
Let me ask , is it neccessary to create a person when we can search for existing person

Reply:
it's ok to select an existing patient

Reply:
This cssSelector element allows input of person name